pulsar-rpc-contrib/src/test/java/org/apache/pulsar/rpc/contrib/PulsarRpcBase.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.pulsar.rpc.contrib;
+
+import static java.util.UUID.randomUUID;
+import java.time.Duration;
+import java.util.function.Supplier;
+import java.util.regex.Pattern;
+import lombok.Setter;
+import org.apache.pulsar.client.admin.PulsarAdmin;
+import org.apache.pulsar.client.api.PulsarClient;
+import org.apache.pulsar.client.api.Schema;
+import org.apache.pulsar.rpc.contrib.client.PulsarRpcClient;
+import org.apache.pulsar.rpc.contrib.server.PulsarRpcServer;
+
+@Setter
+public abstract class PulsarRpcBase {
+    protected final Supplier<String> correlationIdSupplier = () -> randomUUID().toString();
```

Comment:
PulsarRpcBase and SingletonPulsarContainer do not seem to be testcases. It is recommended to change a package, such as org.apache.pulsar.rpc.contrib.base